internal/driver/glfw/window_desktop.go

```diff
@@ -261,26 +261,89 @@ func (w *window) fitContent() {
 	}
 }
 
+// monitorInfo contains the information needed for monitor detection
+type monitorInfo struct {
+	X, Y, Width, Height int
```

Comment:
Best not to make these public fields, even though it's a private structure we go for private by default as a safety / style feature.